Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower_test.py

```diff
@@ -316,3 +343,994 @@ def test_raw_response_to_context_access_policy(list_input, list_output):
 def test_raw_response_to_context_ruls(list_input, list_output):
     result = raw_response_to_context_rules(list_input)
     assert list_output == result
+
+
+''' Helper Functions '''  # pylint: disable=pointless-string-statement
+
+
+def assert_output_has_no_links(outputs: Union[list[dict[str, Any]], dict[str, Any]]):
+    """
+    Check that there are no 'links' keys in the outputs.
+    Args:
+        outputs (list[dict[str, Any]] | dict[str, Any]): output to loop through.
+    """
+    if isinstance(outputs, dict):
+        outputs = [outputs]
+
+    for output in outputs:
+        assert 'links' not in output
+
+
+def assert_command_results(command_results: CommandResults, method: str, expected_output_prefix: str = ''):
+    """
+    Test that the command results outputs has no links in it, if it exists.
+    Test the output prefix.
+    Test readable output.
+
+    Args:
+        command_results (CommandResults): CommandResults created by the tested command.
+        method (str): Key to the expected message within the CommandResults readable output.
+        expected_output_prefix (str): Expected output prefix to be within the CommandsResults.
+            Defaults to ''.
+    """
+    message_by_method = {
+        'POST': 'Created',
+        'GET': 'Fetched',
+        'PUT': 'Updated',
+        'DELETE': 'Deleted',
+    }
+
+    if not command_results.readable_output:
+        assert False
+
+    assert message_by_method[method] in command_results.readable_output
+
+    if method != 'DELETE':
+        context_prefix = '.'.join((INTEGRATION_CONTEXT_NAME, expected_output_prefix))
+
+        assert command_results.outputs_prefix == context_prefix
```

Comment:
please compare the expected output, for each command. See `HelloWorld_test.py`.

Reply:
Added more asserts everywhere 🙂